packages/app/components/Insights/components/SimpleChartComponent.tsx

```diff
@@ -0,0 +1,21 @@
+// import { Bar } from "@ant-design/charts";
+import { SimpleChartOutputType } from "@koh/common";
+import React, { ReactElement } from "react";
+
+interface SimpleChartTypes {
+  output: SimpleChartOutputType;
+}
+
+function SimpleChartComponent({ output }: SimpleChartTypes): ReactElement {
```

Comment:
what's the purpose of this? Is this something for the future?

Reply:
The `SimpleChartComponent` we didn't quite get working because we were getting an error in the node module for @ant-design/charts which we were trying to use. We're still trying to nail down the best charting library to use. 

The reason we left this in here was because we wanted to show an example of how the output of an insight could render a chart.

Reply:
bet